### PR TITLE
Add Highbury Chain

### PR DESCRIPTION
**Pull request by sudopunk**
Add native chain for the Blackfury Ecosystem.

### CODE REVIEW

File: highbury/chain.json [hunk 1]
```diff
@@ -0,0 +1,99 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "highbury",
+  "status": "live",
+  "network_type": "mainnet",
+  "website": "https://www.fury.black/",
+  "pretty_name": "Highbury",
+  "chain_id": "highbury_710-1",
+  "bech32_prefix": "fury",
```
Comment (JeremyParish69) [inline highbury/chain.json:9]: Please register this prefix
https://github.com/satoshilabs/slips/blob/master/slip-0173.md

File: highbury/assetlist.json [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+      "display": "USDF",
+      "symbol": "USDF",
+      "logo_URIs": {
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/highbury/images/usdf.svg",
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/highbury/images/usdf.png"
+      },
+      "coingecko_id": "usdf"
+    },
+    {
+      "denom_units": [
```
Comment (JeremyParish69) [inline highbury/assetlist.json:94]: Pleas remove this asset. I'm guessing it was left here from a copy-paste??

File: highbury/assetlist.json [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+      ],
+      "base": "usdf",
+      "name": "USDF",
+      "display": "USDF",
+      "symbol": "USDF",
+      "logo_URIs": {
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/highbury/images/usdf.svg",
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/highbury/images/usdf.png"
+      },
+      "coingecko_id": "usdf"
```
Comment (JeremyParish69) [inline highbury/assetlist.json:91]: This cgid doesn't match. Please remove

File: highbury/assetlist.json [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+      ],
+      "base": "jinxy",
+      "name": "Jinxy",
+      "display": "JINXy",
+      "symbol": "JINXy",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/highbury/images/jinxy.png",
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/highbury/images/jinxy.svg"
+      },
+      "coingecko_id": "fury-swap"
```
Comment (JeremyParish69) [inline highbury/assetlist.json:69]: This cgid doesn't match. Please remove

File: highbury/assetlist.json [hunk 5]
```diff
@@ -0,0 +1,120 @@
...
+      ],
+      "base": "jinx",
+      "name": "Jinx",
+      "display": "JINX",
+      "symbol": "JINX",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/highbury/images/jinx.png",
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/highbury/images/jinx.svg"
+      },
+      "coingecko_id": "jinx"
```
Comment (JeremyParish69) [inline highbury/assetlist.json:47]: This cgid doesn't exist. Please remove

File: highbury/assetlist.json [hunk 6]
```diff
@@ -0,0 +1,120 @@
...
+      ],
+      "base": "ufury",
+      "name": "Fury",
+      "display": "fury",
+      "symbol": "FURY",
+      "logo_URIs": {
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/highbury/images/fury.svg",
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/highbury/images/fury.png"
+      },
+      "coingecko_id": "fury"
```
Comment (JeremyParish69) [inline highbury/assetlist.json:25]: This cgid doesn't exist. Please remove

File: highbury/assetlist.json [hunk 7]
```diff
@@ -0,0 +1,120 @@
...
+  "assets": [
+    {
+      "description": "The native staking and governance token of the Blackfury ecosystem",
+      "denom_units": [
+        {
+          "denom": "ufury",
+          "exponent": 0
+        },
+        {
+          "denom": "fury",
```
Comment (JeremyParish69) [inline highbury/assetlist.json:13]: What's the relation to fanfury?

Comment (sudopunk) [reply highbury/assetlist.json:13]: Fanfury was originally launched on Terra Classic shortly before the UST depeg, Later the asset was migrated to the Juno network, where there were some internal issues that required a native chain for it to thrive.
Highbury network, will now be the home to the Fanfury project and FURY will be its native asset.

File: highbury/assetlist.json (hunk 6 above)
Comment (sudopunk) [reply highbury/assetlist.json]: This entry has been revised with the correct cgid - "fanfury".

File: highbury/chain.json (hunk 1 above)
Comment (sudopunk) [reply highbury/chain.json:9]: Registration in progress. Please find the active PR in this link:
[https://github.com/satoshilabs/slips/pull/1600](url)

File: highbury/chain.json [hunk 8]
```diff
@@ -0,0 +1,99 @@
...
+  "daemon_name": "fury",
+  "node_home": "$HOME/.fury",
+  "slip44": 459,
```
Comment (JeremyParish69) [inline highbury/chain.json:12]: this is Kava's slip44, while FURY was registered as 710

File: highbury/chain.json [hunk 9]
```diff
@@ -0,0 +1,99 @@
...
+  "alternative_slip44s": [
```
Comment (JeremyParish69) [inline highbury/chain.json:13]: is there an alternate, or was this copied from Kava, too?

File: highbury/assetlist.json (hunk 7 above)
Comment (JeremyParish69) [reply highbury/assetlist.json:13]: Will the Juno variant be decommissioned? Is there any way to exchange juno.FURY to highbury.FURY , or maybe some airdrop?

File: highbury/chain.json (hunk 9 above)
Comment (sudopunk) [reply highbury/chain.json:13]: The alternate has now been updated to 710. Thanks for letting me know.
We intend to follow a similar approach as Irisnet.

File: highbury/chain.json (hunk 8 above)
Comment (sudopunk) [reply highbury/chain.json:12]: Yes, it will be kava's slip44.

File: highbury/assetlist.json (hunk 2 above)
Comment (sudopunk) [reply highbury/assetlist.json:70]: Irrelevant asset removed.

Comment (JeremyParish69) [reply highbury/assetlist.json:70]: I'm referring to this USDT--it's still here...

Comment (sudopunk) [reply highbury/assetlist.json]: That's removed too. Please proceed with your review. Thank you.

**Pull request merged**